packages/ckeditor5-dev-tests/bin/test-travis.sh

```diff
@@ -11,10 +13,9 @@ PACKAGE_NAME=$(node -e "console.log( require( process.cwd() + '/package.json' ).
 # The `.ckeditor5_test_environment` file is created by the "install-dependencies" script.
 CKEDITOR5_TEST_ENVIRONMENT=$(cat ${PACKAGE_ROOT}/.ckeditor5_test_environment)
 
-# Those tasks must be executed from the original package.
+# Linter and dependency checker tasks must be executed from the original package.
 yarn run lint && \
-${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies
-
+${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies && \
```

Comment:
Taking that we have `set -e` it the `&&` still necessary?

Looks safe but might be superfluous...

Reply:
I added two solutions which fix one problem. Most probably we don't need this change but I am safer if we would keep it.